core/src/main/resources/META-INF/operations

```diff
@@ -0,0 +1,29 @@
+# OpenCV Operations
+edu.wpi.grip.core.operations.opencv.MatFieldAccessor
+edu.wpi.grip.core.operations.opencv.NewPointOperation
+edu.wpi.grip.core.operations.opencv.NewSizeOperation
+edu.wpi.grip.core.operations.opencv.MinMaxLoc
+
+# Composite Operations
+edu.wpi.grip.core.operations.composite.NormalizeOperation
+edu.wpi.grip.core.operations.composite.FindContoursOperation
+edu.wpi.grip.core.operations.composite.CascadeClassifierOperation
+edu.wpi.grip.core.operations.composite.HSVThresholdOperation
+edu.wpi.grip.core.operations.composite.ResizeOperation
+edu.wpi.grip.core.operations.composite.BlurOperation
+edu.wpi.grip.core.operations.composite.SwitchOperation
+edu.wpi.grip.core.operations.composite.MaskOperation
+edu.wpi.grip.core.operations.composite.FindBlobsOperation
+edu.wpi.grip.core.operations.composite.HSLThresholdOperation
+edu.wpi.grip.core.operations.composite.ValveOperation
+edu.wpi.grip.core.operations.composite.DesaturateOperation
+edu.wpi.grip.core.operations.composite.RGBThresholdOperation
+edu.wpi.grip.core.operations.composite.FilterContoursOperation
+edu.wpi.grip.core.operations.composite.SaveImageOperation
+edu.wpi.grip.core.operations.composite.ConvexHullsOperation
+edu.wpi.grip.core.operations.composite.FilterLinesOperation
+edu.wpi.grip.core.operations.composite.ThresholdMoving
+edu.wpi.grip.core.operations.composite.PublishVideoOperation
+edu.wpi.grip.core.operations.composite.WatershedOperation
+edu.wpi.grip.core.operations.composite.FindLinesOperation
+edu.wpi.grip.core.operations.composite.DistanceTransformOperation
```

Comment:
Can we get Gradle to build this automatically with some sort of annotation processor?

Reply:
We'd have to write our own annotation processor, and do something for publishables (reports etc) . That'd probably mean creating a new project for the annotations and annotation processor.